pkgs/applications/networking/giara/default.nix

```diff
@@ -54,20 +55,10 @@ python3.pkgs.buildPythonApplication rec {
     python-dateutil
     praw
     pillow
-    mistune
+    mistune_2_0
```

Comment:
mistune_2_0 is only an alias to mistune, so you can revert this line change.

Reply:
Thanks! Must've been looking at the wrong channel. Reverted.